### DISCUSSION TITLE
multiple item loading

### DISCUSSION THREAD

**Discussion by imakihi** [TerriaJS/terriajs#5110] [Ideas] [answered]
Hi there.
I thought it would be really nice if I could select multiple catalog items from the Data Catalogue. Currently I need to open the catalog dialog three times when I want to load three data.

<img width="544" alt="2020-12-22_20-50-42" src="https://user-images.githubusercontent.com/4824436/102960928-ed7bdc00-4497-11eb-89f8-2b1aeb6b43e1.png">

Or, can I configure Terria.io to make it happens?

Hiroo

**Comment by hardreddata** [comment, marked as answer]
Hi @imakihi 

You can hold down the `ctrl` key when you click on the plus buttons to stay in the catalogue.

Enjoy.

**Reply by RichFrazier** [reply]
Thanks for sharing this option.  Ctrl doesn't work on the mac.  But I figured out you can hold down the shift key to make this work.   However, we have a lot of users who are asking for the same feature.  Please consider.  Thank you.
